### DISCUSSION TITLE
Mettre en place un serveur de mail avec OpenSMTPD, Dovecot et Rspamd · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#113] [Articles]

**Comment by djakk** [comment]
Merci !! Je crois que je vais essayer :)

**Comment by Glandos** [comment]
Pour l'apprentissage auto, je conseille https://johannes.sipsolutions.net/Projects/dovecot-antispam/

Je ne peux pas parler de l'installation (c'est empaqueté dans ma distribution), mais à la configuration, ça me semble plus simple que les règles sieve.

**Comment by poolpOrg** [comment]
Le plugin antispam a été déprécié à ma connaissance:

https://dovecot.org/pipermail/dovecot/2017-February/106989.html

A minima sur mon OS, le package a disparu récemment :-/

**Comment by CybsecTn** [comment]
J'ai tenté plusieurs fois, et malgrès que l'on m'as tenu la main à un niveau frisant l’absurdité. Et bien je n'ai pas réussi, pourtant mon hébergeur m'assure avoir autorisé le port 25.

**Comment by HoYoHoYo82** [comment]
Franchement super tutoriel :)
auriez vous un article du même genre pour que nos mail ne tombent pas dans les spam sur des adresse microsoft du style live.fr , outlook.com etc ?
Merci

**Reply by Glandos** [reply]
Ça, c'est le Saint-Graal. La poule aux œufs d'or. Le One Piece.
Dans la vraie vie, malheureusement, ça n'existe pas. Parce que les gros fournisseurs font ce qu'ils veulent, malgré le respect de toutes les règles d'envoi. Le pire étant Hotmail/Live.com qui « accepte » les messages et les mets aux oubliettes, sans en signaler l'expéditeur ou le destinataire. Au moins, GMail les mets en spams…
Ma liste de fournisseurs problématiques :
- GMail
- Outlook/Live.com/Hotmail (c'est Microsoft)
- laposte.net

Ce n'est pas tout le temps facile avec les fournisseurs d'accès en France (SFR, Orange, Free notamment), mais en général, ça passe.

**Comment by caywin25** [comment]
Hello

Merci pour cet article, il est top en revanche, comment on peut faire pour que les mails envoyé depuis des hosts locaux puissent bypass la règle : 

```filter check_rdns phase connect match !rdns disconnect "550 no rDNS is so 80s"```

Merci pour votre aide.
